src/main/java/training/subsystems/wrist/Wrist.java

```diff
@@ -34,7 +34,7 @@ public void goToAngle(Rotation2d targetAngle) {
 	}
 
 	public boolean isAtAngle(Rotation2d targetAngle) {
-		return Math.abs(targetAngle.getDegrees() - motor.getEncoder().getPosition()) == 0;
+		return Math.abs(targetAngle.getDegrees() - motor.getEncoder().getPosition()) == WristConstant.TOLERANCE.getRadians();
```

Comment:
look into motor.getEncoder().getPosition(), it returns rotations, and you compare it to radians, also do <= and not ==
look everywhere if the types ur comparing are the same.
![image](https://github.com/user-attachments/assets/49490882-8890-4c41-b23f-2f8a942d206f)